# HG changeset patch
# User Rik <rik@octave.org>
# Date 1375334027 25200
#      Wed Jul 31 22:13:47 2013 -0700
# Node ID 9d9fe4361870577bd77f875f5873ec7176a12d12
# Parent  1f2c232feaffaa75301e03e9df757f7519cca24b
print.m: Document that '-loose' is default option for eps.

* scripts/plot/print.m: Document that '-loose' is default option for eps.

diff --git a/scripts/plot/print.m b/scripts/plot/print.m
--- a/scripts/plot/print.m
+++ b/scripts/plot/print.m
@@ -211,18 +211,19 @@
 ## @item -append
 ##   Append PostScript or PDF output to a pre-existing file of the same type.
 ##
 ## @item -r@var{NUM}
 ##   Resolution of bitmaps in pixels per inch.  For both metafiles and
 ## SVG the default is the screen resolution; for other formats it is 150 dpi.
 ## To specify screen resolution, use "-r0".
 ##
-## @item -tight
-##   Force a tight bounding box for eps files.
+## @item  -loose
+## @itemx -tight
+##   Force a tight or loose bounding box for eps files.  The default is loose.
 ##
 ## @item -@var{preview}
 ##   Add a preview to eps files.  Supported formats are:
 ##
 ##   @table @code
 ##   @item -interchange
 ##     Provide an interchange preview.
 ##
